services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js

```diff
@@ -193,6 +202,28 @@ export class ProcessPermit extends Component {
       });
   };
 
+  openGeneratePermitNumberModal = () => {
+    this.props
+      .fetchDraftPermitByNOW(
+        this.props.noticeOfWork.mine_guid,
+        this.props.noticeOfWork.now_application_guid
+      )
+      .then(() => {
+        if (this.props.draftPermit.permit_no.includes("DRAFT")) {
```

Comment:
Doesn't the component already have access to this information? Why does it need to be fetched again?

Reply:
chatted in via video. resolved.